jsvg-javafx/src/main/java/com/github/weisj/jsvg/ui/jfx/skin/FXSVGCanvasSkin.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2025 Jannis Weis
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and
+ * associated documentation files (the "Software"), to deal in the Software without restriction,
+ * including without limitation the rights to use, copy, modify, merge, publish, distribute,
+ * sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT
+ * NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+ * DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ */
+package com.github.weisj.jsvg.ui.jfx.skin;
+
+import javafx.animation.*;
+import javafx.scene.control.SkinBase;
+import javafx.scene.layout.StackPane;
+
+import org.jetbrains.annotations.Nullable;
+
+import com.github.weisj.jsvg.SVGDocument;
+import com.github.weisj.jsvg.renderer.animation.AnimationState;
+import com.github.weisj.jsvg.ui.jfx.FXSVGCanvas;
+import com.github.weisj.jsvg.view.ViewBox;
+
+/**
+ * Implementation of the {@link SkinBase} for {@link FXSVGCanvas}, internal use only
+ */
+public class FXSVGCanvasSkin extends SkinBase<FXSVGCanvas> {
+
+    private final FXSVGCanvas svgCanvas;
+    private final StackPane innerPane;
+
+    private final AnimationTimer timer;
```

Comment:
Should animations already work? For me the animation timer stops immediately and no changes in the canvas happen.